### DISCUSSION TITLE
Is there a way to refresh the DB connection?

### DISCUSSION THREAD

**Discussion by lucharo** [tconbeer/harlequin#695] [Q&A, Troubleshooting, Support] [answered]
When dropping tables or schemas, I like to refresh the DB connection to check that indeed the tables/schemas are gone. 

It'd be cool if Cmd+R or similar work to refresh the DB connection!

**Comment by tconbeer** [comment, marked as answer]
`ctrl+r` will refresh the data catalog. See https://harlequin.sh/docs/bindings
